surveys/templates/partials/sidebar.html

```diff
@@ -68,15 +71,29 @@ <h3><b>Just Spaces</b></h3>
             <div><a href="{% url 'signup' %}">Create new account</a></div>
             {% endif %}
             <br />
-            <div><em>Just Spaces</em> is a tool from DataMade and University City District to promote better and more just public spaces</div>
+            <div class="about-blurb"><strong>Just Spaces</strong> is a tool from <a href="https://www.universitycity.org/" target=_blank>University City District</a> and <a href="https://datamade.us/" target=_blank>DataMade</a> to promote better and more just public spaces</div>
         </ul>
     </nav>
 
     <script type="text/javascript">
-        $(document).ready(function () {
-            $('#sidebarCollapse').on('click', function () {
-                $('.sidebar').toggleClass('active');
-                $('.sidebar-blurb').toggleClass('d-none');
-            });
+      var $sidebar = $('.sidebar');
+
+      $(document).ready(function () {
+        resizeSidebar();
+
+        $('#sidebarCollapse').on('click', function () {
+            $sidebar.toggleClass('active');
         });
+      });
+
+      $(window).resize(function() {
+        resizeSidebar();
+      });
+
+      function resizeSidebar() {
+        if (window.innerWidth <= 1000) {
```

Comment:
What's your reasoning behind 1000px as the sidebar breakpoint? Bootstrap 4 uses 992px as the breakpoint for "large screens", that would make sense to me to use here as well: https://getbootstrap.com/docs/4.0/layout/overview/#responsive-breakpoints

Reply:
Just that it felt right! 992px is more sensible.